cmd/lib.go

```diff
@@ -56,28 +66,20 @@ func wrapHost(ctx context.Context, hst host.Host) (host.Host, error) {
 
 func addHostFlagsCommon(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(
-		&ssh, "ssh", "", defaultSsh,
+		&ssh, "target-ssh", "s", defaultSsh,
 		"Applies configuration to given hostname using SSH in the format: [<user>[;fingerprint=<host-key fingerprint>]@]<host>[:<port>]",
 	)
 
 	cmd.Flags().StringVarP(
-		&dockerContainer, "docker-container", "", defaultDockerContainer,
-		"Applies configuration to given Docker container name",
+		&docker, "target-docker", "d", defaultDocker,
+		"Applies configuration to given Docker container name \n"+
+			"Use given format '[<user>]@<container_name>|<container_id>'",
```

Comment:
I think this may need to match the format on `internal/host/docker_unix.go`:

```suggestion
			"Use given format '[<name|uid>[:<group|gid>]@]<image>'",
```